cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -650,3 +643,24 @@ def setup_tokens(token, username, user):
         token = ''.join(random.SystemRandom().choice(alpha) for _ in range(32))
     with open(known_tokens, 'w') as stream:
         stream.write('{0},{1},{2}'.format(token, username, user))
+
+
+def all_kube_system_pods_running():
```

Comment:
I love this! I think there's an opportunity for us to abstract and put this in a kubectl lib or we could potentially use the python kubernetes lib moving forward, but as a straw man, <3 <3 <3